<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-VQJ3RG5" name="com.catchexceptions.seleniumpom.testcases.DataProviderExample" tests="3" failures="0" timestamp="8 Oct 2018 01:29:42 GMT" time="0.007" errors="0">
  <testcase name="verifyData1" time="0.002" classname="com.catchexceptions.seleniumpom.testcases.DataProviderExample"/>
  <testcase name="verifyData1" time="0.003" classname="com.catchexceptions.seleniumpom.testcases.DataProviderExample"/>
  <testcase name="verifyData1" time="0.002" classname="com.catchexceptions.seleniumpom.testcases.DataProviderExample"/>
</testsuite> <!-- com.catchexceptions.seleniumpom.testcases.DataProviderExample -->
